server/plugin/command.go

```diff
@@ -569,8 +580,27 @@ func (p *Plugin) handleWebhookAdd(_ *plugin.Context, parameters []string, args *
 	txt += fmt.Sprintf(" * [%s](%s%d)\n", label, hookURL, *githubHook.ID)
 	return txt
 }
+func (p *Plugin) getOauthTokenScopes(token string) ([]string, error) {
+	var scopes []string
+	req, err := http.NewRequest("HEAD", "https://api.github.com/users/codertocat", nil)
```

Comment:
Is there a more "neutral" API endpoint to use like `https://api.github.com/user`?

Reply:
@hanzei  yes this works `https://api.github.com/user`